lib/helper/ArticleData.dart

```diff
@@ -22,127 +21,109 @@ class _ArticleDataState extends State<ArticleData> {
-                                    )
-                                  : ClipRRect(
-                                      borderRadius: BorderRadius.circular(10.0),
-                                      child: Image(
-                                        image: NetworkImage(
-                                            widget.data[index]['urlToImage']),
-                                      ),
-                                    ),
-                              Expanded(
-                                child: Container(
-                                  height: responsiveTitleHeight(mediaQuery),
-                                  child: Text(
-                                    widget.data[index]['title'] == null
-                                        ? 'Breaking News'
-                                        : widget.data[index]['title'],
-                                    style: GoogleFonts.merriweather(
-                                      textStyle: TextStyle(
-                                          letterSpacing: .4,
-                                          fontSize: 18.0,
-                                          fontWeight: FontWeight.w600),
-                                    ),
-                                  ),
-                                ),
-                              ),
-                              Expanded(
-                                child: Container(
-                                  child: Text(
-                                    widget.data[index]['description'] == null
-                                        ? ''
-                                        : widget.data[index]['description'],
-                                    style: GoogleFonts.merriweather(
-                                      textStyle: TextStyle(
-                                          letterSpacing: .2,
-                                          fontSize: 14.0,
-                                          fontWeight: FontWeight.w400),
-                                    ),
+      child: ListView.builder(
+        physics: const AlwaysScrollableScrollPhysics(),
+        itemCount: widget.data.length,
+        itemBuilder: (context, index) {
+          return GestureDetector(
+            onTap: () {
+              mediaQuery.size.width > 700
+                  ? _launchURL(widget.data[index]['url'])
+                  : Navigator.push(
+                      context,
+                      MaterialPageRoute(
+                        builder: (context) => Details(
+                          widget.data,
+                          index,
+                        ),
+                      ),
+                    );
+            },
+            child: Container(
+              margin: EdgeInsets.only(
+                bottom: 20,
+                left: 4,
+                right: 4,
```

Comment:
Remove left and right margin . This was supposed to be used with the web version

Reply:
done